backend/chromeApiServer/src/main/java/com/canvas/service/helperServices/CanvasClientService.java

```diff
@@ -43,14 +42,14 @@ public class CanvasClientService {
     private static final Logger logger = LoggerFactory.getLogger(CanvasClientService.class);
 
 
-    /**
-     * Constructor with not arguments
-     */
-    public CanvasClientService() {
+    @Autowired
+    protected CanvasClientService(Environment env) {
+        this.canvasUrl = env.getProperty("canvas.url");
         this.okHttpClient = new OkHttpClient();
     }
 
-    protected CanvasClientService(OkHttpClient okHttpClient) {
+    protected CanvasClientService(String canvasUrl, OkHttpClient okHttpClient) {
+        this.canvasUrl = canvasUrl;
```

Comment:
I like that we are using a constant, however this will affect the usability of our app having it referenced the http://10.242.15.70:3000 url, since we don't have a database to fetch files. It makes sense to have this for the Oauth, but we will want to have the others reference either https://seattleu.instructure.com/api/v1 or https://canvas.instructure.com/api/v1 for demoing. Can you update that in this file, since all of them have been updated to use the local canvas. Let me know if that doesn't make sense

Reply:
consider making this the `canvas.oauth.url` and leaving everything else as the original urls we were using.